caml-types.el: Fix missing format argument

hchunhui opened the pull request:
Fixes: 5fa4e02 ("caml-types.el: Extract the feedback logic to a separate function.")

gasche commented:
Indeed, sorry -- I should have reviewed the refactoring more carefully.

Given that the faulty changed is part of the 4.04.0 release, could you please write a Changes entry to let other users know that the issue is fixed? It would be extra nice if you could include in it the error message given by Emacs in the error case (assuming there is an error message).

Please include your Changes entry, respecting the format, in the `Bug fixes` section of the Next minor version (4.04.1)`, so that it is included in the next bugfix release -- if any.

hchunhui committed:
caml-types.el: Fix missing format argument

Now the emacs mode can not show the kind of call at point. Emacs shows the
following message:

  let: Wrong number of arguments: (lambda (info format) "Displays INFO using the given FORMAT." (message (format format info)) (save-current-buffer (set-buffer caml-types-buffer) (erase-buffer) (insert info))), 1

This issue was introduced in commit 5fa4e02. It extracted the feedback logic
to function caml-types-feedback, but deleted the format argument by accident.
So let's fix it.

Fixes: 5fa4e02 ("caml-types.el: Extract the feedback logic to a separate function.")

hchunhui commented:
@gasche Thank you! I have changed my commit.

Now the emacs mode can not show the kind of call at point. Emacs shows the
following message:

  let: Wrong number of arguments: (lambda (info format) "Displays INFO using the given FORMAT." (message (format format info)) (save-current-buffer (set-buffer caml-types-buffer) (erase-buffer) (insert info))), 1

This issue was introduced in commit 5fa4e02. It extracted the feedback logic
to function caml-types-feedback, but deleted the format argument by accident.
So let's fix it.

Fixes: 5fa4e02 ("caml-types.el: Extract the feedback logic to a separate function.")

gasche commented:
Thanks for the nice work! I merged in `trunk` and in the `4.04` release branch.
